server/src/sas/SyntaxDataProvider.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under SAS Code Extension Terms, available at Code_Extension_Agreement.pdf
 
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types,
-@typescript-eslint/no-unused-vars,@typescript-eslint/no-explicit-any */
+@typescript-eslint/no-unused-vars,@typescript-eslint/no-explicit-any,@typescript-eslint/dot-notation */
```

Comment:
@scnwwu Do you know if this file was copy/pasted from somewhere else? In other words, should it be treated as "generated" code? If not, I'm wondering if there's a plan to cleanup the rules we disable here?